### DISCUSSION TITLE
No API Access from Home Assistant

### DISCUSSION THREAD

**Discussion by SandraCHC** [blakeblackshear/frigate#14049] [General Support]
I have installed the Figate Addon in Home Assistant, and make only recordings on Motion.
Now I would like to start a recording from an automation für 30 Seconds.

I have try the API from SSH with CURL

```
~ curl -d "sub_label=lighton&duration=1&include_recording=true" -H "Content-Type: application/x-www-form-urlencoded" -X POST https://localhost:5000/api/events/MyCam/test/create
curl: (7) Failed to connect to localhost port 5000 after 0 ms: Could not connect to server
```

What is my issue that the API is not answering?

**Comment by NickM-27** [comment]
you are using the wrong content type, it should be application/json

**Comment by SandraCHC** [comment]
Thank you @NickM-27 
But unfortunately the issue is the same

```
~ curl -d "sub_label=lighton&duration=1&include_recording=true" -H "Content-Type: application/json" -X POST https://localhost:5000/api/events/MyCam/202409301351-t1/create
curl: (7) Failed to connect to localhost port 5000 after 0 ms: Could not connect to server
```

I have try with http and https, with localhost and 127.0.0.1 and with the local ip from Home Assistant

**Reply by NickM-27** [reply]
Your formatting is wrong, it needs to be json. Besides that, are you sure port 5000 is open

**Comment by SandraCHC** [comment]
Thank you for your realy fast answer @NickM-27 

>> Your formatting is wrong

What do you mean?

>> Besides that, are you sure port 5000 is open

I have installed HA in a Proxmox Container, and have installed the add-on.
My IP in HA is 192.168.1.55, and my FQND https://ha.mydomain.tld:8123
I don't found a setting in the addon to enable the api or the Port 5000
The curl command is used from terminal (SSH Addon) in HA directly.

I also used the blueprint and the Frigate integration to send notifications. The proxied api call with ...8123/api/frigate/notification... is working.

But if iI try https://ha.mydomain.tld:8123/api/frigate/evebts... 
I get a 404: Not found.

**Reply by NickM-27** [reply]
You can't use that, you have to use the 5000 port

**Comment by SandraCHC** [comment]
But why I can't use the API on port 5000 in my HA Terminal?
Is the API only active if I install a separate Container for Frigate and not the Frigate Addon?

**Reply by NickM-27** [reply]
It's disabled by default for the addon hence I'm telling you to enable it

**Reply by SandraCHC** [reply]
Ouch. I not recognize the toggle "show disabled ports..."
Thank you very much

Do you know, If I can send a notification from a recording from API?
